app/assets/stylesheets/application.scss

```diff
@@ -67,13 +65,25 @@ h2 {
     font-size: 22px;
     font-weight: bold;
     font-family: "Roboto", sans-serif;
+    margin-bottom: 0.5em;
 }
 
 h3 {
     font-size: 20px;
     color: $grey-darkest;
     font-weight: 700;
     font-family: "Roboto", sans-serif;
+    margin-bottom: 0.5em;
+}
+
+h2 span,
+h3 span {
+    display: block;
```

Comment:
Not sure if we want to add this CSS to a top-level selector. Maybe a class for the span would be a better option?

Reply:
Agree. @dburka what would you call these little heading spans?